editor/code_editor.cpp

```diff
@@ -358,14 +362,21 @@ void FindReplaceBar::_update_results_count() {
 	}
 }
 
-void FindReplaceBar::_update_matches_label() {
+void FindReplaceBar::_update_matches_label(TextEditSearchResult* p_search_result = nullptr) {
 	if (search_text->get_text().is_empty() || results_count == -1) {
 		matches_label->hide();
 	} else {
 		matches_label->show();
 
 		matches_label->add_theme_color_override("font_color", results_count > 0 ? get_theme_color(SNAME("font_color"), SNAME("Label")) : get_theme_color(SNAME("error_color"), SNAME("Editor")));
-		matches_label->set_text(vformat(results_count == 1 ? TTR("%d match.") : TTR("%d matches."), results_count));
+		if (p_search_result)
+			matches_label->set_text(vformat(
+				results_count == 1 ? TTR("%d match.") : TTR("%d/%d matches."),
```

Comment:
Suggested wording for better readability:

```suggestion
				results_count == 1 ? TTR("%d match") : TTR("%d of %d matches"),
```

Visual Studio Code:

![image](https://user-images.githubusercontent.com/180032/162637742-64aefb1b-e23a-4563-a481-be85fdf8390e.png)

Firefox:

![image](https://user-images.githubusercontent.com/180032/162637750-cd41cdbf-0bf2-4b83-b5c6-ddf5072df723.png)